client/components/project-form.jsx

```diff
@@ -43,7 +43,7 @@ export default function ProjectForm() {
       },
       body: JSON.stringify({ poNumber, name, address, city, state, zipcode, notes })
     };
-    fetch('/api/project/newproject', req)
+    fetch('/api/projects/newproject', req)
```

Comment:
As I mentioned before, every time I read this I wonder why `newproject` is part of the endpoint.